.github/workflows/wheel_manylinux_pypi.yaml

```diff
@@ -10,7 +10,7 @@ on:
     branches:
       - main
   schedule:
-    - cron: '0 5 * * *' # 5 AM UTC
+    - cron: '0 5 * 3 *' # Every Wed 5 AM UTC
```

Comment:
i think this is supposed to be 
```suggestion
    - cron: '0 5 * * 3' # Every Wed 5 AM UTC
```
according to [cronitor](https://crontab.guru/#0_5_*_*_3) but then i had a vague memory GH Actions uses a different cron format but i couldn't find it after a minute or two of searching. mind confirming for me?

Reply:
There is a tip: You could use Github online editor for any workflow file. When moving cursor over the cron, Github editor will interpret the cron for you. For example:

![image](https://user-images.githubusercontent.com/8262694/176975398-36867eef-22c0-4ee2-8bed-e07eddbc1c25.png)

Reply:
Yeah, that's correct @areusch. I merged your suggestion, thanks!

Reply:
thanks for the tip @comaniac ! did not know that and it's very useful.